### DISCUSSION TITLE
Scadenza e segnalazioni cancellate

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3870] [General]
Buongiorno,
credo di aver mal interpretato quanto in oggetto.
Pensavo che una segnalazione arrivata a scadenza (Es. dopo 90 giorni), venisse _taggata_ come Scaduta ma non eliminata.
Invece la data di scadenza presuppone la cancellazione (se ho capito correttamente).
Pertanto avendo l'obbligo di mantenere le segnalazioni per 5 anni a partire dall'apertura, ha senso che metta come **Scadenza (giorni)** il valore 1825 (ovvero 365 * 5, o 1826)?
E come **Data del promemoria (giorni)** che metta 87 (90 giorni della scadenza - 3 giorni di avviso _preventivo_)?
Mediamente come vengono impostati questi campi per essere adempienti con la normativa? 
Scusate ma ripeto, avevo totalmente mal interpretato questi campi.
Ma poi, le segnalazioni cancellate, lo sono definitivamente e via email e/o nei log non è possibile averne evidenza?
Grazie

**Comment by evilaliv3** [comment]
Buongiorno,

Non esiste obbligo di mantenere le segnalazioni per 5 anni apartire dall'apertura. Dove legge questa indicazione?
L'obbligo è di non tenerle oltre 5 anni dalla chiusura.

In merito alla data di promemoria si va bene quanto propost, eventualmente anche un promemoria a 60 giorni può essere utile.

Nelle nostre configurazioni impostiamo 12 mesi di data retention e 60 giorni di promemoria.

Per le segnalazioni cancellate viene mantenuto un log in audit log visibile agli amministratori del sistema; c'è previsione di estendere questo audit log anche ai riceventi.

**Reply by maelaborazioni** [reply]
Sì, chiedo scusa, mi sono espresso male circa la normativa. 
Grazie per aver chiarito questi miei dubbi!
